### PR TITLE
[#IOCIT-22] Update io-backend Dockerfile node version

### PR DESCRIPTION
**Pull request by arcogabbo**
#### List of Changes
Updated the node version pulled from docker

#### Motivation and Context
Container build failing cause of node version requirements

#### How Has This Been Tested?
Container builded correctly

#### Types of changes

- [ ] Chore (nothing changes by a user perspective)
- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

#### Checklist:

- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

File: io-backend/Dockerfile [hunk 1]
```diff
@@ -10,15 +10,15 @@ ARG GITHUB_TOKEN
...
 
 RUN rm -rf * && \
-    git clone https://github.com/pagopa/io-backend.git /usr/src/app && \
-    git checkout $IO_BACKEND_COMMIT_SHA
+  git clone https://github.com/pagopa/io-backend.git /usr/src/app && \
+  git checkout $IO_BACKEND_COMMIT_SHA
 
 RUN ./scripts/init-npm.sh
 
 RUN yarn install && \
```
Comment (balanza) [inline io-backend/Dockerfile:18]: should we add `--frozen-lockfile?`

Comment (michaeldisaro) [reply io-backend/Dockerfile:18]: It could be a container more similar to production. I agree.

Comment (balanza) [reply io-backend/Dockerfile:18]: The point is not `yarn.lock` file to change, the point is the container to run with **exactly same packages** at each installation

Comment (arcogabbo) [reply io-backend/Dockerfile]: Applied the change in fde259b .
I will open a PR for other containers so we can make them more stable.

**Pull request merged**